### ISSUE TITLE
Twitter icon is failing contrast - WCAG 2.1 1.4.11

### ISSUE THREAD

**Issue by nfrenette** [wet-boew/wet-boew#9280]
Twitter icon in .followus section is failing contrast - WCAG 2.1 1.4.11

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://www.canada.ca/en/canadian-heritage.html
2. Test the contrast ratio of the twitter image against the 'followus' class background colour (#eaebed)

**Who shall do the work?**
I am asking for Principal publisher to please do the work

**Screenshots**
![image](https://user-images.githubusercontent.com/15619641/153865139-74858fed-8d7d-4f4f-97b1-c20c189f9b7c.png)

**Comment by kodz1** [comment]
Twitter officially offers a black logo https://about.twitter.com/en/who-we-are/brand-toolkit, would this pass?

Here the current official logos:
![2021 Twitter logo - black](https://user-images.githubusercontent.com/23484460/155727436-2ba501c5-bc3d-46b1-b3be-e1999b39ae5f.png)
![2021 Twitter logo - blue](https://user-images.githubusercontent.com/23484460/155727438-a77c25fa-1e15-4879-becd-9410037632e9.png)
![2021 Twitter logo - white](https://user-images.githubusercontent.com/23484460/155727439-5738ec00-6743-40a0-aa70-64377b8f9421.png)

**Comment by duboisp** [comment]
FYI - @delisma
